cmd/backend/main.go

```diff
@@ -138,9 +138,9 @@ func main() {
 	unitService := unit.NewService(logger, dbPool)
 	distributeService := distribute.NewService(logger, unitService)
 	formService := form.NewService(logger, dbPool)
-	questionService := question.NewService(logger, dbPool)
+	questionService := question.NewService(logger, question.New(dbPool))
 	inboxService := inbox.NewService(logger, dbPool)
-	responseService := response.NewService(logger, dbPool)
+	responseService := response.NewService(logger, response.New(dbPool))
```

Comment:
Consider to revert it and implement it in `internal/form/question/service.go`
```suggestion
	responseService := response.NewService(logger, dbPool)
```